ipaserver/install/server/install.py

```diff
@@ -940,7 +940,29 @@ def install(installer):
 
     print("======================================="
           "=======================================")
-    print("Setup complete")
+    print("IPA server installation successful")
+    print("")
+    print("You can access Web UI using https://%s/ipa/ui" % (host_name))
+    print("")
+    print("Obtain a Kerberos Ticket for issuing IPA command 'kinit admin'.")
+    print("")
+    print("Server is installed with following roles:")
+    print("* 389 Directory Server")
```

Comment:
If you insist on printing all roles they should be indented.

Reply:
I cannot understand your comment.
On console it prints as:

Server is installed with following roles:
* 389 Directory Server
* Kerberos Server
* HTTP Server
* CA Server
* DNS Server